content/en/platform/corda/4.3/enterprise/cryptoservice-configuration.md

```diff
@@ -289,11 +288,7 @@ or if gradle is not on the path but gradlew is in the current directory then run
 ./gradlew shadowJar
 ```
 
-This will create a jar called azure-keyvault-with-deps.jar, copy this into the drivers directory.
-
-Note that okhttp3 needs to be shaded as the latest version of this library will raise an exception on a handle leak which version 1.2.1
```

Comment:
We need to be careful here.

We shouldn't be removing `relocate 'okhttp3', 'shadow.okhttp3'` from docs like that without 
making sure QA verified it would work.

The associated ticket was mostly talking about removing the "Note that okhttp3 ..." sentence from the versions, where the `relocate 'okhttp3', 'shadow.okhttp3'` part is actually missing. Having a quick look at our docs, this seems to be the case only for 4.8

Reply:
In that case this bug was addressed in DOC-3662. I shall close this PR as no change is required.